### PR TITLE
History

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
added history page with JSX

### CODE REVIEW

File: src/pages/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+.home-heading {
+  text-align: center;
+  margin: 2.4rem;
+  margin-top: 3rem;
+  font-size: 3rem;
+}
+
+.home-heading span {
+  color: var(--primary);
+  font-family: "Nunito sans", cursive;
```
Comment (tejas-muthya37) [inline src/pages/Home/Home.css:10]: Store fonts in CSS variables to make them reusable.

File: src/pages/Home/Home.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  z-index: 10;
+  transition: all 0.3s ease-in-out;
+}
+
+.home-hero-button:hover .btn-text {
+  font-weight: 500;
+}
+
+.shopNow {
+  color: #2d97c6;
```
Comment (tejas-muthya37) [inline src/pages/Home/Home.css:61]: Again, store colors in CSS variables.

**Review by tejas-muthya37 — APPROVED** [review]
I have reviewed the History component of this PR since that's what this PR was intended for. Please keep your PRs small.

**Review by Hnh013 — APPROVED** [review]
code looks structured and clean ,
a suggestion , if feasible use variable for css properties if they are called multiple times , as tejas mentioned

File: src/pages/Home/Home.css (hunk 2 above)
Comment (Aman-Jain1996) [reply src/pages/Home/Home.css:61]: This value is used here only so didn't stored in variable

File: src/pages/Home/Home.css (hunk 1 above)
Comment (Aman-Jain1996) [reply src/pages/Home/Home.css:10]: Sure tejas , will do it

**Pull request merged**